tests/readonly_app/models.py

```diff
@@ -1,10 +1,10 @@
 from __future__ import unicode_literals
```

Comment:
Is it necessary now?
I think it's also worth to check all `__future__` imports in the code

Reply:
Done